### PR TITLE
fixed position issue

### PR DESCRIPTION
**Pull request by VIRAT9358**
#496  fixed
Custom location navigator position is changed from 'fixed' to 'relative' .
Code changes-
if (style.position === 'static') { style.position = 'relative'; }

### CODE REVIEW

**Review by iangilman — CHANGES_REQUESTED** [review]
Apologies... I appear to have led you to the wrong spot! This file doesn't need to change. Instead here's the spot that needs changing: 

https://github.com/openseadragon/openseadragon/blob/17cd6b51ce34fbf3ac3fdcce86851cccb1b12f8b/src/controldock.js#L62

Also, instead of looking at the element's style directly, we should get the computed style (so we pick up any styling that came from CSS style sheets), like so:

```
$.getElementStyle(this.element).position
```

One more thing: the statement needs to be broken up so the `if` and its opening curly bracket are on one line, the assignment is on a second line, and the ending curly bracket is on the third line. Otherwise our linter won't allow it to pass.

Thank you for working on this!

File: src/controldock.js [hunk 1]
```diff
@@ -59,7 +59,10 @@
         if( this.element ){
             this.element = $.getElement( this.element );
             this.element.appendChild( this.container );
-            this.element.style.position = 'relative';
+            // this.element.style.position = 'relative';
```
Comment (iangilman) [inline src/controldock.js:62]: Please delete this... we don't need it commented out here.

File: src/navigator.js [hunk 2]
```diff
@@ -162,7 +162,7 @@ $.Navigator = function( options ){
     this.displayRegion.className = 'displayregion';
 
     (function( style, borderWidth ){
-        style.position      = 'relative';
+        style.position = 'relative';
```
Comment (iangilman) [inline src/navigator.js:165]: Please update this so it matches the original. As you can see here, you've removed some spaces next to the equals sign.

**Comment by VIRAT9358** [comment]
Changes made as mentioned by you .
Regards ,
Virat Srivastava

**Review by iangilman — APPROVED** [review]
Beautiful! Thank you for solving this issue and taking the time to get it right!

**Pull request merged**